src/components/inspector/controls/FormControl.tsx

```diff
@@ -1,24 +1,22 @@
 import React, { ReactNode, memo } from 'react'
 import { useSelector } from 'react-redux'
-import { FiRepeat } from 'react-icons/fi'
 import {
   FormControl as ChakraFormControl,
   Grid,
   Box,
   FormLabel,
   Text,
 } from '@chakra-ui/core'
-import { SmallCloseIcon } from '@chakra-ui/icons'
 
 import PopOverControl from './PopOverControl'
 import {
-  getShowCustomComponentPage,
   isInstanceOfCustomComponent,
   getSelectedComponentId,
   getPropsOfSelectedComp,
 } from '../../../core/selectors/components'
-import ActionButton from '../ActionButton'
-import useDispatch from '../../../hooks/useDispatch'
+import UnExposePropButton from '../../actionButtons/UnExposePropButton'
+import CustomPropDeletionButton from '../../actionButtons/CustomPropDeletionButton'
+import { checkIsCustomPage } from '../../../core/selectors/page'
```

Comment:
The logic in this component is way more than what a FormControl component needs to do.


For example: this component acts differently if the form control is representing the props of a Custom Component, or the props a child components of a custom component which are exposed.


Ideally this should have been two different components itself